doc/classes/PackedVector2Array.xml

```diff
@@ -52,7 +52,7 @@
 			<param index="1" name="before" type="bool" default="true" />
 			<description>
 				Finds the index of an existing value (or the insertion index that maintains sorting order, if the value is not yet present in the array) using binary search. Optionally, a [param before] specifier can be passed. If [code]false[/code], the returned index comes after all existing entries of the value in the array.
-				[b]Note:[/b] Calling [method bsearch] on an unsorted array results in unexpected behavior.
+				[b]Note:[/b] Calling [method bsearch] on an unsorted array results in unexpected behavior. [constant @GDScript.NAN] doesn't behave the same as other numbers.
```

Comment:
I would make this two "Notes" instead of one. 
```suggestion
				[b]Note:[/b] Calling [method bsearch] on an unsorted array results in unexpected behavior.
				[b]Note:[/b] [constant @GDScript.NAN] doesn't behave the same as other numbers.
```

Additionally, the comment may not work in this context. Vector2 isn't really a number, so are you talking about if the Vector2 contains an NaN, or are you saying that an element could return an NaN instead of a Vector2?

Reply:
Thank you will look into it for vector specifically, this is about vectors with elements that are `NaN`

Reply:
How does "Vectors with elements that are NAN doesn't behave the same as other Vectors" sound?

Reply:
Its better (although it should say "don't" instead of "doesn't" as "Vectors" is plural). 

I think what is missing is the "so what?" factor. It is one thing to point out that NaNs won't be treated the same as other numbers, but what happens if an NaN sneaks into the input to this method? 

If we are looking for a phrase that can be reused across the various methods here maybe something like:
"[b]Note: [/b] Vectors with elements that are NAN don't behave the same as other Vectors. Therefore, the results from this method may not be accurate if NaNs are included."

Reply:
That sounds better thank you, will amend

Reply:
Forgot the operators in Vector and will amend there too